### PR TITLE
feat: traffic analysis prevention using fixed headers size

### PR DESCRIPTION
**Pull request by marcouberti**
## Description

- changes the default headers to make them consistent across all the network calls

## Checklist

- [x] I have followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md).
- [x] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [ ] I have written new tests for my core changes, as applicable.
- [x] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

### CODE REVIEW

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰


✅ All configuration files are in line with the master branch.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #212 (id: caf954f4-ab0b-464e-999b-c6c0b1886d13)
- Last verified commit: #6523d21180e103ec7187c5c50a03242b2c7acbdc
- Time of check: 26/05/2020, 13:45:36 UTC
- The following protected files have been checked for changes: app/proguard-rules.pro, fastlane/Fastfile, .gitmodules, ci/danger/commitlint/index.ts, app/build.gradle, yarn.lock, package.json, Gemfile, gradlew.bat, ci/danger/ktlint.ts, .editorconfig, dangerfile.ts, Gemfile.lock, template.properties, ci/danger/commitlint/rules.ts, gradlew, gradle.properties, ci/danger/danger.ts, build.gradle, ci/changelog_config.tpl, settings.gradle, fastlane/Appfile, scheduler_config.json, publiccode.yml.

File: extensions/src/main/java/it/ministerodellasalute/immuni/extensions/http/TrafficAnalysisPreventionHeadersInterceptor.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import java.io.IOException
+
+/**
+ * TrafficAnalysisPreventionHeadersInterceptor is a [OkHttpClient] network interceptor that
+ * remove some default added http headers.
```
Comment (madbat) [inline extensions/src/main/java/it/ministerodellasalute/immuni/extensions/http/TrafficAnalysisPreventionHeadersInterceptor.kt:26]: I'd change `remove some default added http headers` to `removes some of the http headers added by default`

**Comment by ImmuniDGR** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **extensions/http/TrafficAnalysisPreventionHeadersInterceptor.kt#L18** - ktlint (0.36.0)<br>parent path: extensions/src/main/java/it/ministerodellasalute/immuni<br><br>Issue: Imports must be ordered in lexicographic order without any empty lines in-between (rule: import-ordering)
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: Commitlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Thank you for submitting a pull request! The team will review your submission as soon as possible.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 132f93258b18922b8adcebcb71bae5cbe3caa0c3
</p>

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰


✅ All configuration files are in line with the master branch.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #257 (id: 88a4a3dd-11e8-4904-b1fb-4ffeb6b9cc31)
- Last verified commit: #4b00c8f0e0c1c83d3b925d73cffc5f1a7d23be7f
- Time of check: 26/05/2020, 22:01:16 UTC
- The following protected files have been checked for changes: .editorconfig, publiccode.yml, gradlew.bat, Gemfile, Gemfile.lock, scheduler_config.json, yarn.lock, ci/danger/danger.ts, build.gradle, ci/changelog_config.tpl, settings.gradle, app/proguard-rules.pro, fastlane/Appfile, gradlew, ci/danger/commitlint/rules.ts, dangerfile.ts, package.json, .gitmodules, gradle.properties, fastlane/Fastfile, ci/danger/commitlint/index.ts, app/build.gradle, ci/danger/ktlint.ts, template.properties.

**Pull request merged**